jekyll/_cci2/ip-ranges.md

```diff
@@ -100,13 +100,17 @@ IP address ranges for core services (used to trigger jobs, exchange information
 
 - 18.214.70.5
 - 52.20.166.242
-- 35.174.249.131
 - 18.214.156.84
 - 54.236.156.101
+- 52.22.215.219
+- 52.206.105.184
+- 52.6.77.249 
+- 34.197.216.176
+- 35.174.249.131
 - 3.210.128.175
 
-**Upcoming Changes to list of IP address ranges** (Last Updated: 08/02/2021):
-None
+**Upcoming Changes to list of IP address ranges** (Last Updated: 08/23/2021):
```

Comment:
This feels like it should have semantic structure rather than just being bolded. This will enable us to build a structured list of changes.

I'm a little concerned about calling these "upcoming changes," as it could be confusing. Maybe there should be more clarity about what the 30-day notice period will mean.

```suggestion
### Upcoming changes to the list of IP address ranges
```